openedx/features/content_tagging/rest_api/v1/views.py

```diff
@@ -0,0 +1,155 @@
+        object (default: False).
+        * allow_multiple (optional): Indicates that multiple tags from this taxonomy may be added to
+        an object (default: False).
+        * allow_free_text (optional): Indicates that tags in this taxonomy need not be predefined; authors
+        may enter their own tag values (default: False).
+
+    **Create Example Requests**
+        POST api/tagging/v1/taxonomy                                                - Create a taxonomy
+        {
+            "name": "Taxonomy Name",                    - User-facing label used when applying tags from this taxonomy to Open edX objects."
+            "description": "This is a description",
+            "enabled": True,
+            "required": True,
+            "allow_multiple": True,
+            "allow_free_text": True,
+        }
+
+
+    **Create Query Returns**
+        * 201 - Success
+        * 403 - Permission denied
+
+    **Update Parameters**
+        * id (required): - The id of the taxonomy to update
+
+    **Update Request Body**
+        * name (optional): User-facing label used when applying tags from this taxonomy to Open edX objects.
+        * description (optional): Provides extra information for the user when applying tags from this taxonomy
+        to an object.
+        * enabled (optional): Only enabled taxonomies will be shown to authors.
+        * required (optional): Indicates that one or more tags from this taxonomy must be added to an object.
+        * allow_multiple (optional): Indicates that multiple tags from this taxonomy may be added to an object.
+        * allow_free_text (optional): Indicates that tags in this taxonomy need not be predefined; authors may
+        enter their own tag values.
+
+    **Update Example Requests**
+        PUT api/tagging/v1/taxonomy/:id                                            - Update a taxonomy
+        {
+            "name": "Taxonomy New Name",
+            "description": "This is a new description",
+            "enabled": False,
+            "required": False,
+            "allow_multiple": False,
+            "allow_free_text": True,
+        }
+        PATCH api/tagging/v1/taxonomy/:id                                          - Partially update a taxonomy
+        {
+            "name": "Taxonomy New Name",
+        }
+
+    **Update Query Returns**
+        * 200 - Success
+        * 403 - Permission denied
+
+    **Delete Parameters**
+        * id (required): - The id of the taxonomy to delete
+
+    **Delete Example Requests**
+        DELETE api/tagging/v1/taxonomy/:id                                         - Delete a taxonomy
+
+    **Delete Query Returns**
+        * 200 - Success
+        * 404 - Taxonomy not found
+        * 403 - Permission denied
+
+    """
+
+    class TaxonomyPagination(DefaultPagination):
+        page_size = 100
+        max_page_size = 1000
+
+    pagination_class = TaxonomyPagination
```

Comment:
I'm sorry, didn't notice that the pagination was only added here instead of on the openedx-learning side..
Do you mind to create a small PR against openedx-learning that moves this over there?

Reply:
Done: https://github.com/openedx/openedx-learning/pull/69

Reply:
ToDo: remove pagination here

Reply:
Your PR is merged and now available with `openedx-learning==0.1.2`.

If you run `make requirements` in your LMS shell, that should bump the installed version in the `requirements/edx/*` files.
And if you run into this issue, see [these comments on stackoverflow](https://stackoverflow.com/q/76585758):
```
Exception: Can not find valid pkg-config name.
      Specify MYSQLCLIENT_CFLAGS and MYSQLCLIENT_LDFLAGS env vars manually
```